README.md

```diff
@@ -83,27 +79,80 @@ The core set of binding data is:
 * **password** - the password or token used to log into the service.  Can be omitted if no authorization required, or take another format such as an API key.  It is strongly recommended that the corresponding ConfigMap metadata properly describes this key.
 * **certificate** - the certificate used by the client to connect to the service.  Can be omitted if no certificate is required, or simply point to another Secret that holds the client certificate.  
```

Comment:
> ... or simply point to another Secret that holds the client certificate.

Not related to changes in this PR, but was thinking how to differentiate between a certificate value or a secret?

Reply:
I was thinking the metadata (or at least documentation) would be the key to describe the content - similar as to how `password` could be a basic auth pass, an apiKey, a token, etc.